docs/components/text-input.md

```diff
@@ -53,3 +53,23 @@ Error messages for text input components are configurable in Umbraco on the sett
 For a `[RegisteredCharityNumber]` attribute set the 'Text input width' setting to 'small' and use the 'Pattern' error message.
 
 ![Settings for a postcode using a text input component](/docs/images/registered-charity-number-settings.png)
+
+
+## Allowed file fypes
```

Comment:
Looks like this is a draft you forgot to remove?

Reply:
Removed the AllowedFileType section from TextInput.md